exercises/practice/protein-translation/protein-translation.jl

```diff
@@ -0,0 +1,3 @@
+function rna_translator(rna::String)
```

Comment:
Function names should describe an action, not an object, according to the Julia style guide and general practice.

How about one of these names? Presented in my preference order.

```suggestion
function rna_to_amino_acids(rna::String)
```

```suggestion
function rna_to_protein(rna::String)
```

```suggestion
function translate_rna(rna::String)
```